cmd/mattermost/commands/team_test.go

```diff
@@ -67,14 +63,10 @@ func TestLeaveTeam(t *testing.T) {
 
 	}
 
-	if found {
-		t.Fatal("profile should not be on team")
-	}
+	require.False(t, found, "profile should not be on team")
 
 	if teams, err := th.App.Srv.Store.Team().GetTeamsByUserId(th.BasicUser.Id); err != nil {
-		if len(teams) > 0 {
-			t.Fatal("Shouldn't be in team")
-		}
+		require.Equal(t, 0, len(teams), "Shouldn't be in team")
 	}
```

Comment:
I suggest to have nil check on err as well, like
```go
teams, err := th.App.Srv.Store.Team().GetTeamsByUserId(th.BasicUser.Id)
require.Nil(t, err)
require.Equal(t, 0, len(teams), "Shouldn't be in team hello")
```

Reply:
agree, I'll make an update